Projects/hCoreUI/Sources/hForm/Gradient/GradientState.swift

```diff
@@ -0,0 +1,33 @@
+import Flow
+import Foundation
+import SwiftUI
+import UIKit
```

Comment:
Seen a few places where `UIKit` is still imported even though it's not used. It's not going to effect compile time or anything but would later help us identify all the places where `UIKit` is still being used.

Reply:
You're right, I'll try to remove all such occurrences!